cloverage/src/cloverage/report/html.clj

```diff
@@ -3,18 +3,22 @@
    [clojure.string :as cs]
    [clojure.java.io :as io]
    [cloverage.source :refer [resource-reader]]
-   [cloverage.report :refer [line-stats total-stats file-stats with-out-writer]])
+   [cloverage.report :refer [line-stats total-stats file-stats with-out-writer]]
+   [hiccup.core :as h]
+   [hiccup.page :as page])
   (:import
    [java.io File]))
 
-(def html-escapes
-  {\space "&nbsp;"
-   \& "&amp;"
-   \< "&lt;"
-   \> "&gt;"
-   \" "&quot;"
-   \' "&#x27;"
-   \/ "&#x2F;"})
+(def JQUERY-SORTING "jquery.tablesorter.min.js")
+(def JQUERY "jquery-3.3.1.min.js")
+(def START "start.js")
+(def STYLESHEET "coverage.css")
+(defn css [& args]
+  (->> args
+       (into {})
+       (map (fn [[k v]]
+              (str (name k) \: v)))
+       (cs/join \;)))
```

Comment:
I can't follow how this works. You appear to call the function with a map, but this takes `& args`, so now `args` is `({:width 1})` or whatever. Why is that something you can pass to `(into {})`?